src/test/java/organice/logic/commands/CommandTestUtil.java

```diff
@@ -82,6 +82,9 @@
     public static final String VALID_DOCTOR_IN_CHARGE_PATIENT_BOB = "S1111111A";
     public static final String VALID_DOCTOR_IN_CHARGE_PATIENT_IRENE = "S1231231B";
 
+    public static final String VALID_STATUS_DONOR_JOHN = "not processing";
+    public static final String VALID_STATUS_PATIENT_IRENE = "not processing";
```

Comment:
Could be changed to Status.STATUS_NOT_PROCESSING

Reply:
amended